ground/src/main/java/com/google/android/ground/ui/home/AboutFragment.kt

```diff
@@ -72,7 +73,8 @@ class AboutFragment : AbstractFragment() {
         Image(
           bitmap = R.drawable.ground_logo.toImageBitmap(),
           contentDescription = "Logo",
-          modifier = Modifier.padding(0.dp, 8.dp, 0.dp, 22.dp).width(122.dp).height(122.dp),
+          modifier =
+            Modifier.testTag("logo").padding(0.dp, 8.dp, 0.dp, 22.dp).width(122.dp).height(122.dp),
```

Comment:
Can we use contentDescription instead of "testTag"?

Reply:
Done